experimental/traffic-portal/src/app/core/statuses/status-details/status-details.component.html

```diff
@@ -0,0 +1,26 @@
+<mat-card class="statuses-details">
+    <ul class="breadcrumb">
+        <li><a routerLink="/core/statuses">Statuses</a></li>
+        <li>{{title}}</li>
+    </ul>
+    <tp-loading *ngIf="loading"></tp-loading>
+    <form [formGroup]="statusDetailsForm" (ngSubmit)="onSubmit()" *ngIf="!loading">
+        <mat-card-content>
+            <mat-form-field class="example-full-width">
+                <mat-label>Name</mat-label>
+                <input matInput placeholder="Ex. Pizza" formControlName="name">
```

Comment:
"Ex." is short for "Exercise", so you probably meant "E.g.". However, it's redundant in any case, because the purpose of a placeholder is to provide an example of valid input, so "E.g." would be implied. I'm also not sure how much information "Pizza" can really be conveying as an example here, especially since it's used for both the name and the description.

Reply:
Shouldn't this be a `required` input?

Reply:
My mistake, i forgot to make appropirate corrections after copying code from material reference.